projects/http-auth/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"html"
+	"log"
+	"net/http"
+	"os"
+	"github.com/joho/godotenv"
+	"golang.org/x/time/rate"
+)
+
+func main() {
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatal("error loading env file")
```

Comment:
You probably want to include the error in here, so that people can try to debug what went wrong.